modules/ROOT/pages/security.adoc

```diff
@@ -66,6 +72,28 @@ Use if you authenticate to your API using custom header parameters and values.
 
 Any other authentication methods to access your added API, such as custom OAuth implementations or short-lived token methods, are not supported.
 
+== Configuring Mutual Authentication in DataGraph
+
+Mutual authentication (mTLs) ensures that Datagraph and the underlying APIs in your unified schema can create secure, verified connections.
+
+If an API requires mTLS, you must configure mTLS when adding it to the unified schema, or when promoting it to another environment. To do so, you must provide the following:
+
+* A keystore that contains:
+** A public certificate signed by your organization's root certificate authority (CA).
+** The private key used to generate the certificate signing request (CSR).
+** The password for the encrypted private key.
+* Optionally, a truststore that contains the root CA.
+
+// [add screenshot]
+
+Anypoint Platform stores keystore and truststore in xref:anypoint-security::index-secrets-manager.adoc[Secrets Manager] in PEM files.
```

Comment:
```suggestion
Anypoint Platform stores the keystore and the truststore in PEM files in the xref:anypoint-security::index-secrets-manager.adoc[secrets manager].
```
secrets manager, the secrets manager
Not a title or product. Never refer to as secrets vault, CSM, or the Secrets Manager.